charts/ingresslibchart/templates/_frontendconfig.yaml

```diff
@@ -0,0 +1,16 @@
+{{- define "libchart.frontendconfig.tpl" -}}
+apiVersion: networking.gke.io/v1beta1
+kind: FrontendConfig
+metadata:
+  name: {{ .Chart.Name  }}-ingress-frontendconfig
+  labels:
+    {{- include "helm.labels" . | nindent 4 }}
```

Comment:
We'll want to include the labels of the parent chart here, so maybe it makes sense to pass that in as a parameter? The liquibase migration chart has a good example for how to do this: https://github.com/broadinstitute/terra-helm/blob/master/charts/liquibase-migration/templates/_job.yaml#L7

Reply:
I noticed that the parent label across charts is repeated and has the same definitions. In general, if additional information needs to be added, they can be as seen [here](https://github.com/broadinstitute/terra-helm/blob/08c549ab09e7904ad37ec8db6d9f89daf25e5164/charts/consent/templates/ingress.yaml#L4). Also, if there are repeated values between the charts, the parent label will take precedence. Hence, there's no need to pass it as a parameter.

Reply:
I think this bit in particular is split across multiple PRs--https://github.com/broadinstitute/terra-helm/pull/422#discussion_r675713160 also notes this

Reply:
I think I understand the merge mechanic now, but I think the problem remains if an app uses different or fewer labels than what is defined by `helm.labels`. If we pass by parameter, it works as expected; if this chart also repeats some expectations about labels, then the end labels become a union of the base chart and what this library provides.

I'm totally with you on the labels being frequently repeated, and I think a library chart is a great way to reduce that repetition--I just don't think that an ingress library chart that isn't common across all apps is the right place to do that abstraction. I think we have an opportunity here to _not_ write down another repetition of the labels, and we can do the pass-by-reference trick to make it easier to abstract in the future.